tools/generate_tables_sql_and_testNames.php

```diff
@@ -1,118 +1,213 @@
 #!/usr/bin/php
 <?php
-
 /**
- * generate_tables_sql.php takes the ip_output.txt file generated from
- * quickform_parser.php and outputs an sql build file for the table of each
- * instrument it finds in the ip_output.txt file.  These sql files are output
- * to the tables_sql/ subdirectory.
+ * Behavioural
  *
- * ex cmd:  php generate_tables_sql.php
+ * PHP Version 7
  *
- * @package behavioural
+ * The generate_tables_sql_and_testNames.php takes the {instrument_name}.linst file
+ * from the /instrument_manager  module and installs an instrument for the table of each
+ * instrument it finds in the {instrument_name}.linst file.
+ * These sql files are output to the tables_sql/ subdirectory.
+ *
+ * Ex cmd with SQL execution:
+ * php generate_tables_sql_and_testNames.php ../instruments/[instrument_name].linst
+ *
+ * Ex cmd without SQL execution:
+ * php generate_tables_sql_and_testNames.php
+ * ../instruments/[instrument_name].linst false
```

Comment:
Change this to be consistent with other scripts:
* Should just print usage by default
* --to-sql flag
* --apply flag